app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,33 @@
+
+<div class='form-page'>
+  <h2>Log in</h2>
+  <div class="form-container">
+    <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+      <% flash.each do |name, message| %>
+          <div class="<%= name %>"><%= message %></div>
+      <% end %>
+      
+      <div class="field">
+        <%= f.label :email %><br />
+        <%= f.email_field :email, autofocus: true, autocomplete: "email", class: 'input' %>
+      </div>
+
+      <div class="field">
+        <%= f.label :password %><br />
+        <%= f.password_field :password, autocomplete: "current-password", class: 'input' %>
+      </div>
+
+      <% if devise_mapping.rememberable? %>
+        <div class="checkbox">
+          <%= f.check_box :remember_me %>
+          <%= f.label :remember_me %>
+        </div>
+      <% end %>
+
+      <div class="actions">
+        <%= f.submit "Log in", class:'create-btn' %>
+      </div>
+    <% end %>
```

Comment:
- [ ] I would recommend to follow the design and color same as given [here](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=).

### Your design

![Screenshot from 2021-12-04 17-48-26](https://user-images.githubusercontent.com/22434660/144709166-31d40cb3-5e10-423c-8851-6a7fd4a96683.png)

### GIven design 

![Screenshot from 2021-12-04 17-51-38](https://user-images.githubusercontent.com/22434660/144709219-b1b983bc-26af-48d7-b68c-9483768aa0c5.png)